### ISSUE TITLE
Blocks don't render properly in the superflat options when not in a loaded world

### ISSUE THREAD

**Issue by LordQuadrato** [ImpactDevelopment/ImpactClient#605]
### What happens?
In the vanilla GUI for creating a superflat preset blocks are not rendered properly (seems like the top and front face are missing)
> Possibly relates to #533 _(Tile entities not shown in Xray menu)_
> Forge could also have a fix for this like for #533

![2018-06-01_18 09 24](https://user-images.githubusercontent.com/38622942/40851588-2fb211cc-65c8-11e8-8d07-de580a7c0d5f.png)

### Exact steps to reproduce?
1. Launch Impact
2. Create a new world and set the type to superflat
Do **NOT** enter a world before
3. Press Customize. You will see that the two front faces of full all blocks aren't rendered.
Issue is not present in a vanilla client.

### Additional Client Info
- Impact Version: 4.2 for 1.12.2
- Using Optifine? No
- Enabled mods: -
- Operating System: Windows 10 64bit

**Comment by LeafHacker** [comment]
Just to confirm, you don't experience without Impact?

**Comment by LordQuadrato** [comment]
Yes without Impact everything is rendered correctly. I already added the information in the steps to reproduce:
> Issue is not present in a vanilla client.
